src/style.js

```diff
@@ -72,7 +72,7 @@ export default (editor, opt = {}) => {
           }],
       }, {
         name: 'Decorations',
-        open: false,
+        open: true,
         buildProps: ['background-color', 'container-background-color', 'background-url', 'background-repeat',
           'background-size', 'border-radius', 'border'],
         properties: [{
```

Comment:
Any issue reference to this change? 
IMHO is better to have less UI open, but if this is something highly requested I can accept it otherwise I'd prefer a rollback

Reply:
For me, i think it is better that the default is close.
I prefer the rollback for this.

Reply:
This one also come from one of my users that complained too many clicks required to change styles. The default to close is okay.